vector/v.net/v.net.html

```diff
@@ -252,8 +252,3 @@ <h3>TURNS SUPPORT</h3>
 Implementation: Stepan Turek 
 Documentation: Lukas Bocan
 Mentor: Martin Landa
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 98 

```html
<em><a href="v.net.alloc.html">v.net.alloc</a></em>,
<em><a href="v.net.iso.html">v.net.iso</a></em>,
<em><a href="v.net.path.html">v.net.path</a></em>,
<em><a href="v.net.salesman.html">v.net.salesman</a></em>
```

should be 

```html
<em>
  <a href="v.net.alloc.html">v.net.alloc</a>,
  <a href="v.net.iso.html">v.net.iso</a>,
  <a href="v.net.path.html">v.net.path</a>,
  <a href="v.net.salesman.html">v.net.salesman</a>
</em>
```

Line 126

```html
<em><a href="v.net.allpairs.html">v.net.allpairs</a></em>, 
<em><a href="v.net.bridge.html">v.net.bridge</a></em>, 
<em><a href="v.net.centrality.html">v.net.centrality</a></em>, 
<em><a href="v.net.components.html">v.net.components</a></em>, 
<em><a href="v.net.distance.html">v.net.distance</a></em>, 
<em><a href="v.net.flow.html">v.net.flow</a></em>, 
<em><a href="v.net.spanningtree.html">v.net.spanningtree</a></em>, 
<em><a href="v.net.steiner.html">v.net.steiner</a></em>, 
<em><a href="v.net.timetable.html">v.net.timetable</a></em>, and 
<em><a href="v.net.visibility.html">v.net.visibility</a></em>.
```

should be 

```html
<em>
  <a href="v.net.allpairs.html">v.net.allpairs</a>, 
  <a href="v.net.bridge.html">v.net.bridge</a>, 
  <a href="v.net.centrality.html">v.net.centrality</a>, 
  <a href="v.net.components.html">v.net.components</a>, 
  <a href="v.net.distance.html">v.net.distance</a>, 
  <a href="v.net.flow.html">v.net.flow</a>, 
  <a href="v.net.spanningtree.html">v.net.spanningtree</a>, 
  <a href="v.net.steiner.html">v.net.steiner</a>, 
  <a href="v.net.timetable.html">v.net.timetable</a>, 
  <a href="v.net.visibility.html">v.net.visibility</a>
</em>.
```

Line 231

 `<a href="v.net.salesman.html">v.net.salesman</a>` 

should be 

`<a href="v.net.salesman.html">v.net.salesman</a>,`